apps/overlay/src/views/lowerthirds/CustomCarousel.o.tsx

```diff
@@ -1,20 +1,39 @@
 import { FC, PropsWithChildren } from "react"
-import { motion } from "framer-motion"
+import { motion, Transition, Variant } from "framer-motion"
+interface CustomCarouselProps extends PropsWithChildren {
+  animation?: AnimationProps
+  transition?: Transition
+}
+
+type AnimationProps = {
+  [key: string]: Variant
+}
```

Comment:
This can be proved but okay for now.

this can be a `Record<"initial" | "animate" | "exit", Variant>`

or can be imported from `framer-motion`